snews.php

```diff
@@ -206,7 +206,6 @@ function checkUserPass($input) {
 function readAddons() {
 	static $admin_mods;
 	if (!$admin_mods) {
-		global $l;
```

Comment:
You cannot delete global $l (you loose lang definition from addons).
Try to use hello addon like
http://localhost/snews1.80/hello/